s3am/Makefile

```diff
@@ -0,0 +1,25 @@
+# Definitions
+build_tool = runtime-container.DONE
+name = quay.io/ucsc_cgl/s3am
+tag = 1.0a1
```

Comment:
When installing via pip from PyPI, no upstream commit should be necessary. The package version on PyPI contains a build number, e.g. in 1.0b1.dev28 this is build 28 on Jenkins. The Jenkins build references the commit. IOW, everything can still be followed back to a commit.

Tagging the image in quay.io with the cgl-docker-lib commit allows for incremental builds. Without that we would have to build and push (!) every tool, every time.

If at all possible, PRs should be consistent with the current method of tagging. I'd rather have an inferior method across the board, than a mix of methods, some inferior, some superior. I'm open to switching methods but would prefer to do so consistently, for all tools, at one time.